packages/components/pie-webc-core/src/test-helpers/percy-lit-options.ts

```diff
@@ -0,0 +1,37 @@
+///*
+///* This code snippet was provided by Percy as a way to get our Lit components working correctly when running against the Safari browser.
+///* Safari blocks requests coming from localhost, resulting in our CSS not loading when requested.
+///* This function is passed to Percy and executed against the serialised DOM to rewrite all 'localhost' urls to 'render.percy.local'.
+///* This is a temporary workaround, and a long-term solution will be implemented by Percy at a later date.
+///*
+const percyOptions = {
+    domTransformation: (documentElement: any) => {
+        function updateLinks(root : any) {
+        root.querySelectorAll('[data-percy-adopted-stylesheets-serialized]').forEach((link : any) => {
+            console.log(link);
+            let href = link.getAttribute('data-percy-serialized-attribute-href');
+            href = href.replace(/localhost[:\d+]*/, 'render.percy.local');
+            link.setAttribute('data-percy-serialized-attribute-href', href);
+        });
+
+        root.querySelectorAll('[data-percy-shadow-host]')
+            .forEach((shadowHost : any) => {
+                console.log(shadowHost);
```

Comment:
Ditto to the above

Reply:
@sebastienrocks I believe we were given this code to use by Percy. If you don't mind I'm going to merge it and we can fix forward if we want to remove them. They're only running during tests so I think it's pretty harmless